mobile/Dockerfile

```diff
@@ -29,18 +29,11 @@ ENV PATH ${PATH}:/opt/android/tools/bin
 RUN sdkmanager --update \
     && yes | sdkmanager \
     "build-tools;26.0.1" \
-    "platforms;android-25"
+    "platforms;android-26"
```

Comment:
This seems inconsistent with earlier notes about using API version 23

Reply:
That is required. I think it's the platform tools version, but not necessarily the API you are building for.
Using '25' or '23' fails the `Cordova requirements`